From f3ca5b14f2eb0b22c802ffa2ff052d1e79c5d3b9 Mon Sep 17 00:00:00 2001
From: Robby Cai <R63905@freescale.com>
Date: Tue, 16 Oct 2012 16:21:28 +0800
Subject: ENGR00229803-2 sii902x: add dependency on CONFIG_FB_MXC_ELCDIF_FB

Add dependency on CONFIG_FB_MXC_ELCDIF_FB, to avoid build error if as module.

Signed-off-by: Robby Cai <R63905@freescale.com>
---
 drivers/video/mxc/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/mxc/Kconfig b/drivers/video/mxc/Kconfig
index 8a4a079..83b09426 100644
--- a/drivers/video/mxc/Kconfig
+++ b/drivers/video/mxc/Kconfig
@@ -61,7 +61,7 @@ config FB_MXC_SII902X
 	tristate "Si Image SII9022 DVI/HDMI Interface Chip"
 
 config FB_MXC_SII902X_ELCDIF
-	depends on FB_MXC_SYNC_PANEL && I2C
+	depends on FB_MXC_ELCDIF_FB && FB_MXC_SYNC_PANEL && I2C
 	tristate "Si Image SII9022 DVI/HDMI Interface Chip for ELCDIF FB"
 
 config FB_MXC_CH7026
-- 
1.7.9.5

